examples/pypi_wheel_provider.py

```diff
@@ -3,6 +3,7 @@
 from io import BytesIO
 from operator import attrgetter
 from platform import python_version
+from sys import argv
```

Comment:
It is a common pattern to only import sys in main if possible.

Reply:
Thanks, TIL.